mcp/run_tools.md

```diff
@@ -73,14 +88,85 @@ To set variables using a `.env` file:
 1. Optionally, set other variables.
 1. Save the file.
 
+## Use the Gemini CLI extension
+
+**Additional prerequisites** 
+In addition to the [standard prerequisites](#prerequisites), you must have the following installed:
+- [Git](https://git-scm.com/)
+- [Google Gemini CLI](https://github.com/google-gemini/gemini-cli#quick-install)
+
+When you install the extension, it clones the [Data Commons extension Github repo](https://github.com/gemini-cli-extensions/datacommons) to your local system.
+
+### Install
+ 
+Open a new terminal and install the extension directly from GitHub:
+```sh
+gemini extensions install https://github.com/gemini-cli-extensions/datacommons
+```
+> Note: If you have previously configured Gemini CLI to use the Data Commons MCP Server and want to use the extension instead, be sure to delete the `datacommons-mcp` section from the relevant `settings.json` file (e.g. `~/.gemini/settings.json`).
+
+### Run
+
+1. From any directory, run `gemini`. 
+1. To verify that the Data commons tools are running, enter `/mcp list`. You should see `datacommons-mcp` listed as `Ready`. If you don't, see the [Troubleshoot](#troubleshoot) section.
+1. To verify that the extension is running, enter `/extensions list`. You should see `datacommons` listed as `active`. 
+1. Start sending [natural-language queries](#sample-queries).
+
+### Update
+
+After starting up Gemini CLI, you may see the message `You have one extension with an update available`.
+
+In this case, run `/extensions list`. If `datacommons` is displayed with `update available`, run the following command:
+```
+gemini extensions update datacommons
```

Comment:
this can either be performed on the terminal with the command you've provided, or it can be updated **within a GCLI session** with 

`/extensions update datacommons`

since the previous command was "/extensions list" within a GCLI session, it might make sense to list this one or both options